easybuild/easyconfigs/c/cdat/cdms2-3.1.5-intel-2020a.patch

```diff
@@ -0,0 +1,16 @@
+diff -ru cdms-3.1.5/Src/Cdunifmodule.c cdms-3.1.5-edit/Src/Cdunifmodule.c
```

Comment:
Please add patch description + link to upstream source or mention patch author (+ fix SHA256 for this patch file)